lib/moon/design/button/utils.ex

```diff
@@ -17,6 +17,16 @@ defmodule Moon.Design.Button.Utils do
     end
   end
 
+  def get_icon_button_size_classes(size) do
```

Comment:
please move it to IconButton module and make it private (defp) if it's used only there